### PR TITLE
Add support for shortening GitHub issues links

### PR DESCRIPTION
**Pull request by Sherlouk**
Relates to #411

### CODE REVIEW

**Comment by Sherlouk** [comment]
btw sorry @rnystrom, couple prs to review now 😅 

Probably want to do a big merge before v1.11

**Comment by rnystrom** [comment]
@Sherlouk wanna add any unit tests for this?

**Comment by Sherlouk** [comment]
Sure, on it now!

**Comment by Sherlouk** [comment]
Should be all good!

File: FreetimeTests/MMMarkdownASTTests.swift [hunk 1]
```diff
@@ -124,5 +124,33 @@ final class MMMarkdownASTTests: XCTestCase {
...
+        let resultTwo = CreateCommentModels(markdown: testTwo, width: 300, options: options)
+        let textTwo = resultTwo.first as! NSAttributedStringSizing
+        XCTAssertEqual(textTwo.attributedText.string, "differentOwner/differentRepo#321")
+        XCTAssertNotNil(textTwo.attributedText.attributes(at: 1, effectiveRange: nil)[MarkdownAttribute.issue])
+        
+        let detailsTwo = textTwo.attributedText.attributes(at: 1, effectiveRange: nil)[MarkdownAttribute.issue] as! IssueDetailsModel
+        XCTAssertEqual(detailsTwo.owner, "differentOwner")
+        XCTAssertEqual(detailsTwo.repo, "differentRepo")
+        XCTAssertEqual(detailsTwo.number, 321)
+    }
```
Comment (rnystrom) [inline FreetimeTests/MMMarkdownASTTests.swift]: ❤️

**Pull request merged**